tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -1,22 +1,22 @@
-# How to Use Ramdump
-Here we explain how to enable build support for RAMDUMP upload,  
+# How to Use Dump Tool
+Here we explain how to enable build support for DUMP upload,  
 and how to actually upload and parse it in the event of a crash during runtime.
 
 ## Contents
-> [Build Steps](#how-to-enable-ramdump)  
-> [Upload Steps](#how-to-upload-ramdump)  
+> [Build Steps](#how-to-enable-dump-tool)  
+> [Upload Steps](#how-to-upload-dump)  
 > [Parsing Steps](#how-to-parse-ramdump)  
-> [Porting Guide](#how-to-port-ramdump)
+> [Porting Guide](#how-to-port-dump-tool)
 
-## How to enable RAMDUMP
-Below configuration must be enabled to support ramdump upload  
+## How to enable DUMP Tool
```

Comment:
It's the enabling way of memory dump, not tool.
Let's say `how to enable memory dump`.

Reply:
Updated